original/pong/main.py

```diff
@@ -85,11 +91,25 @@ def get(self):
         users = session.query(models.User).all()
         wins = [util.user_wins(u.name, session) for u in users]
         template = JINJA_ENVIRONMENT.get_template('users.html')
-        self.response.write(template.render({
+        content = template.render({
             'users': zip(users, wins),
             'form': forms.AddUser(),
-            'logout_url': usersapi.create_logout_url('/users')
-        }))
+            'logout_url': usersapi.create_logout_url('/users'),
+            'is_admin_user': usersapi.is_current_user_admin()
+        })
+        self.response.write(content)
+
+    def post(self):
+        form = forms.AddUser(formdata=self.request.POST)
+        if form.validate():
+            print("FORM VALID")
```

Comment:
Copy pasta from the existing [add game](https://github.com/DanielSank/pong/blob/master/original/pong/main.py#L66) api.